src/lib/dom-api.html

```diff
@@ -593,6 +593,55 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+          var trueDocument = ownerDocument || this.node;
+
+          // Native, non-document ShadowRoot?
+          if (this.node.host && ownerDocument && !window.ShadowDOMPolyfill && !this.node._isShadyRoot) {
+            return this.node.activeElement;
+          }
+
+          // Not ShadowRoot-like?
+          if (!this.node.host && ownerDocument) {
+            return this.node.activeElement;
+          }
+
+          var active = trueDocument.activeElement;
+          if (!active) return null;
+
+          if (window.ShadowDOMPolyfill) {
+            active = ShadowDOMPolyfill.wrap(active);
+          }
+
+          var activeRoot = Polymer.dom(active).getOwnerRoot();
+          while (activeRoot && activeRoot !== this.node) {
+            active = activeRoot.host;
+            activeRoot = Polymer.dom(active).getOwnerRoot();
+          }
+
+          return (!ownerDocument ? !activeRoot : activeRoot === this.node) ? active : null;
+        },
+        set: function(value) {
```

Comment:
For elements that don't have a special activeElement property, this just lets it work as a normal property accessor (the default behavior). It definitely feels like an unlikely use case though, cleanliness is probably more important than correctness here. Think I should remove it?